### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EvgenyOrekhov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # reactions
+test
```
Comment (EvgenyOrekhov) [inline README.md:2]: test

Comment (EvgenyOrekhov) [inline README.md:2]: conversation

Comment (EvgenyOrekhov) [reply README.md:2]: reply to conversation

Comment (EvgenyOrekhov) [reply README.md:2]: reply to test

**Review by EvgenyOrekhov — COMMENTED** [review]
Green button comment

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 # reactions
```
Comment (EvgenyOrekhov) [inline README.md:1]: new

Comment (ArmiT) [inline README.md:1]: test

**Review by ArmiT — COMMENTED** [review]
my comment

File: README.md (hunk 1 above)
Comment (ArmiT) [reply README.md:2]: test #2

Comment (ArmiT) [reply README.md:2]: test #2

File: README.md (hunk 2 above)
Comment (EvgenyOrekhov) [reply README.md:1]: test reply

File: README.md (hunk 1 above)
Comment (EvgenyOrekhov) [reply README.md:2]: test 2 reply

**Review by EvgenyOrekhov — COMMENTED** [review]
green button 2

File: README.md (hunk 1 above)
Comment (EvgenyOrekhov) [reply README.md:2]: single comment

**Pull request closed without merging**